notifiers/providers/victorops.py

```diff
@@ -0,0 +1,91 @@
+from ..core import Provider
+from ..core import Response
+from ..utils import requests
+
+
+class VictorOps(Provider):
+    """Send VictorOps webhook notifications"""
+
+    base_url = "https://portal.victorops.com/ui/{ORGANIZATION_ID}/incidents"
+    site_url = "https://portal.victorops.com/dash/{ORGANIZATION_ID}#/advanced/rest"
+    name = "victorops"
+
+    _required = {
+        "required": [
+            "rest_url",
+            "message_type",
+            "entity_id",
+            "entity_display_name",
+            "message",
+        ]
+    }
+    _schema = {
+        "type": "object",
+        "properties": {
+            "rest_url": {
+                "type": "string",
+                "format": "uri",
+                "title": "the REST URL to use with routing_key. create one in victorops `integrations` tab.",
+            },
+            "message_type": {
+                "type": "string",
+                "title": "severity level can be: "
+                "- critical or warning: Triggers an incident "
+                "- acknowledgement: sends Acknowledgment to an incident "
```

Comment:
just noticed it, you didn't mention `warning` or `ok` in here (and they are included in the enum)

Reply:
`warning` and `critical` are together in the same line because they both will create an "incident" recourse in Victorops (the severity doesn't affect what happens with the alert).

The same goes for `recovery` and `ok`